homework21/Product.java

```diff
@@ -0,0 +1,55 @@
+import java.time.LocalDateTime;
+
+public class Product {
+    private String typ;
```

Comment:
Make the type enum instead of a raw string.

Reply:
+++